2022-01-26 11:34:23,380 INFO    MainThread:15862 [wandb_setup.py:_flush():71] setting env: {}
2022-01-26 11:34:23,380 INFO    MainThread:15862 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-26 11:34:23,381 INFO    MainThread:15862 [wandb_init.py:_log_setup():371] Logging user logs to /home/robotlabx/Desktop/DTAI/decnet2022/src/models/wandb/run-20220126_113423-2iwm6zuj/logs/debug.log
2022-01-26 11:34:23,381 INFO    MainThread:15862 [wandb_init.py:_log_setup():372] Logging internal logs to /home/robotlabx/Desktop/DTAI/decnet2022/src/models/wandb/run-20220126_113423-2iwm6zuj/logs/debug-internal.log
2022-01-26 11:34:23,381 INFO    MainThread:15862 [wandb_init.py:init():404] calling init triggers
2022-01-26 11:34:23,381 INFO    MainThread:15862 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-01-26 11:34:23,381 INFO    MainThread:15862 [wandb_init.py:init():460] starting backend
2022-01-26 11:34:23,381 INFO    MainThread:15862 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-26 11:34:23,399 INFO    MainThread:15862 [backend.py:ensure_launched():216] starting backend process...
2022-01-26 11:34:23,414 INFO    MainThread:15862 [backend.py:ensure_launched():222] started backend process with pid: 15911
2022-01-26 11:34:23,415 INFO    MainThread:15862 [wandb_init.py:init():469] backend started and connected
2022-01-26 11:34:23,417 INFO    MainThread:15862 [wandb_init.py:init():533] updated telemetry
2022-01-26 11:34:23,456 INFO    MainThread:15862 [wandb_init.py:init():563] communicating current version
2022-01-26 11:34:24,020 INFO    MainThread:15862 [wandb_init.py:init():568] got version response 
2022-01-26 11:34:24,020 INFO    MainThread:15862 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-26 11:34:24,245 INFO    MainThread:15862 [wandb_init.py:init():606] starting run threads in backend
2022-01-26 11:34:25,735 INFO    MainThread:15862 [wandb_run.py:_console_start():1810] atexit reg
2022-01-26 11:34:25,735 INFO    MainThread:15862 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-26 11:34:25,735 INFO    MainThread:15862 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-26 11:34:25,736 INFO    MainThread:15862 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-26 11:34:25,736 INFO    MainThread:15862 [wandb_init.py:init():633] run started, returning control to user process
2022-01-26 11:34:25,959 INFO    MainThread:15862 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-26 11:34:25,959 INFO    MainThread:15862 [wandb_run.py:_restore():1752] restore
2022-01-26 11:34:28,258 INFO    MainThread:15862 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 790
}

2022-01-26 11:34:28,543 INFO    MainThread:15862 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 790
}

2022-01-26 11:34:29,255 INFO    MainThread:15862 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 2
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 816
}

2022-01-26 11:34:29,358 INFO    MainThread:15862 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 13405
}

2022-01-26 11:34:29,461 INFO    MainThread:15862 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 13405
}

2022-01-26 11:34:29,564 INFO    MainThread:15862 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1085
  total_bytes: 13405
}

2022-01-26 11:34:29,667 INFO    MainThread:15862 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13405
  total_bytes: 13405
}

2022-01-26 11:34:29,769 INFO    MainThread:15862 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13405
  total_bytes: 13405
}

2022-01-26 11:34:29,875 INFO    MainThread:15862 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13405
  total_bytes: 13405
}

2022-01-26 11:34:29,977 INFO    MainThread:15862 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13405
  total_bytes: 13405
}

2022-01-26 11:34:30,079 INFO    MainThread:15862 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13405
  total_bytes: 13405
}

2022-01-26 11:34:30,605 INFO    MainThread:15862 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13405
  total_bytes: 13405
}

2022-01-26 11:34:31,145 INFO    MainThread:15862 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13405
  total_bytes: 13405
}
local_info {
}

2022-01-26 11:34:32,217 INFO    MainThread:15862 [wandb_run.py:_append_files():2180] logging synced files
